### ISSUE TITLE
Could you provide a download link for the vDiT model and its corresponding test code?

### ISSUE THREAD

**Issue by Buddingpopp** [ByteDance-Seed/ByteCheckpoint#16]
Besides the Training Code Example (https://github.com/ByteDance-Seed/ByteCheckpoint/blob/main/demo/fsdp_save_reshard.py), are there any other codes that can be used to test checkpoint saving on a single machine? The checkpoint generated in the Training Code Example is only a few tens of MB. I want to test vDiT mentioned in the paper or other large models. Could you provide a download link for the vDiT model and its corresponding test code?
